### PR TITLE
DDF-2375 Updated documentation to provide an example WMTS configuration

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR updates the Standard Search UI Imagery Provider documentation to include an example configuration of WMTS/
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@ricklarsen 
@troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire 
#### How should this be tested?
Build the documentation and observe the changes in the managing ui section.
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-2375
#### Screenshots (if appropriate)
https://codice.atlassian.net/browse/DDF-2375
#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by jlcsmith** [comment]
👍

**Comment by ricklarsen** [comment]
👍

File: distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc [hunk 1]
```diff
@@ -173,14 +173,14 @@ Admin Console.
...
  -WMT (WebMapTile), +
  -TMS (TileMapService), +
- -GE (GoogleEarth).
- -
- -Example: TYPE={url=
- -{"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}}
+ -GE (GoogleEarth), +
+ Examples: +
+ -{"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5} +
+ -{"type" "WMT" "url" "http://example.com" "layer" "layer1" "style" "default" "tileMatrixSetID" "tileSetId"}
```
Comment (kcwire) [inline distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc]: Should we just provide an example of all of them?
@codice/ui - will all of these work on both maps (openlayers & ceisum)?

Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc]: I can't answer the second question, but yes we should have examples of all of them.  I'd be OK with doing this in a separate PR though.

Comment (bdeining) [inline distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc]: I also agree that there should be additional examples in the documentation.  These JSON strings are designed to work with both maps ( the examples provided are supported by both APIs).

Comment (kcwire) [inline distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc]: @bdeining are you planning on opening a new PR or another ticket for these?

Comment (bdeining) [inline distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc]: I'll make a ticket for that.

**Pull request merged**